_data/accessibility-tests/date-range-picker.yml

```
@@ -0,0 +1,131 @@
+title: Date range picker
+component_status: pass
+test_items:
+# General tests
+  - summary: Touch target size for all clickable elements meets height and width requirements.
+    summary_additional: |
+      When you select any date range picker element to inspect and review component code,
+      you can see the total width of the touch target is at least 24 pixels wide and 24 pixels tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 2.5.8
+  - summary: Content doesn't change before submission.
+    summary_additional: |
+      When you enter a date into the date range picker,
+      the date stays the same and content doesn't change unless you take another action (e.g., you hit the submit button)
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.2.2
+  - summary: Errors are clearly identified.
+    summary_additional: |
+      If you type in a date range in an incorrect format, the error is identified and described to you in the text.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.1
+  - summary: Text instructions for the date range picker are clear.
+    summary_additional: |
+      When you view the date range picker, you find clear, simple instructions in text for what type of date format is expected.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.2
```

I noticed this was `passes with exception` in our spreadsheet. Should we update here and add the exception explanation?

```suggestion
  - summary: Text instructions for the date range picker are clear.
    summary_additional: |
      When you view the date range picker, you find clear, simple instructions in text for what type of date format is expected.
    test_status: exception
    test_type: general
    status_details: |
      The date range picker component may not pass all elements of this test.
      The helper text relates to what format is expected of the date, but there are not visible instructions for how to use the calendar button. To meet user needs, you will need to add instructions for each element in the date picker.
    github_issue_number: 6241
    github_issue_repo: uswds
    version_tested: 3.7.1
    wcag_criterion: 3.3.2
```

---

I think we're covering that issue in this [conditional formatting work](https://github.com/uswds/uswds/issues/5693)

The exception would be "Some users may not find the instructions clear. Follow our work to improve this" and then link to the above PR. Open to suggestions though from @alex-hull and @jaclinec as I think this arose from some of our usability testing findings too.

---

Updated in [4db550a](https://github.com/uswds/uswds-site/pull/3052/commits/4db550a1c1ac86904559efba099aad4c2811e850)

---

Please confirm that I added the correct "status details" summary here.

---

Lgtm!